src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -344,11 +344,11 @@ private static Version GetMSBuildVersionInPath()
         /// <returns>The matching toolset.</returns>
         /// <remarks>This method is not intended to be called directly. It's marked public so that it
         /// can be called by unit tests.</remarks>
-        public static Toolset SelectMsbuildToolset(
+        public static Tuple<string, string> SelectMsbuildToolset(
```

Comment:
We should have our own type here, that has named properties (maybe private class?) instead of `Item1` and `Item2`.